app/src/main/java/no/usn/mob3000/App.kt

```diff
@@ -118,7 +119,7 @@ val LocalNavController = compositionLocalOf<NavHostController> { error("No NavCo
  * @see Viewport
  * @see ScreenIcon
  * @see Icon
- * @author frigvid
+ * @author Anarox
```

Comment:
You are not the only author of the Viewport. You've contributed to the route definition, this is enough to append yourself as `@contributor Anarox`, but you have not re-written the Viewport, and as such cannot be the author. Revert the change, and add yourself as a contributor.